pkg/tagger/local/tagger.go

```diff
@@ -272,9 +273,26 @@ IterCollectors:
 	}
 	t.RUnlock()
 
-	computedTags := utils.ConcatenateTags(tagArrays)
+	return utils.ConcatenateTags(tagArrays), nil
+}
+
+// TagBuilder appends tags for a given entity from the tagger to the TagsBuilder
+func (t *Tagger) TagBuilder(entity string, cardinality collectors.TagCardinality, tb *util.TagsBuilder) error {
+	tags, err := t.getTags(entity, cardinality)
+	if err == nil {
+		tb.Append(tags...)
```

Comment:
I curious if you know if `func Append(tags []string)` and `func Append(tags ...string)` provide the same  performances.

Did you choose `func Append(tags ...string)` for usage simplicity?

Reply:
I chose it to match the `append` keyword. It's also useful when you want to add only one element, you don't have to allocate a slice for it.